subgraphs/compound-forks/cream-finance/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "private": true,
```

Comment:
what is this for?

Reply:
without this I wasn't able to run the scripts -- on my end scripts either have to have that tag or have the license and all the other headers

Reply:
what is the error log without "private"?

Reply:
well i guess it does work (i just get a license warning). I think my file tree wasn't updating right away. 

Removed